mods/xpanes/init.lua

```diff
@@ -149,9 +147,10 @@ end
 
 xpanes.register_pane("pane", {
 	description = "Glass Pane",
-	textures = {"default_glass.png","xpanes_pane_half.png","xpanes_white.png"},
-	inventory_image = "default_glass.png",
-	wield_image = "default_glass.png",
+	textures = {"default_glass.png^default_glass_detail.png","xpanes_pane_half.png","xpanes_white.png"},
```

Comment:
Why is 'glass detail' needed?

Reply:
ugh, sorry I will fix both of these. I got pulled away and it threw me off focus